snuba/datasets/configuration/events_analytics_platform/storages/uptime_monitor_checks.yaml

```diff
@@ -0,0 +1,61 @@
+version: v1
+kind: writable_storage
+name: uptime_monitor_checks
+storage:
+  key: uptime_monitor_checks
+  set_key: uptime_monitor_checks
+readiness_state: limited
+schema:
+  columns:
+    [
+      { name: organization_id, type: UInt, args: { size: 64 } },
+      { name: project_id, type: UInt, args: { size: 64 } },
+      { name: environment, type: String, args: { schema_modifiers: [nullable, low_cardinality] } },
```

Comment:
this should not be low cardinality, it's set by a customer, you can't guarantee it

Reply:
environment is set to low cardinality on some other datasets, was just copying based on that. i can remove it if you'd like though though https://github.com/getsentry/snuba/blob/4c7d8d90fa0186a3b2de9039a79a3196878ed079/snuba/datasets/configuration/issues/entities/search_issues.yaml#L25 https://github.com/getsentry/snuba/blob/4c7d8d90fa0186a3b2de9039a79a3196878ed079/snuba/snuba_migrations/events/0011_rebuild_errors.py#L24